SorterOfPersonsByAge/src/PersonSorter.java

```diff
@@ -0,0 +1,39 @@
+public class PersonSorter {
+    Persons [] personsToSort;
+
+    public PersonSorter(Persons[] personsArrayToSort) {
```

Comment:
constructors should be used for constructing objects only and not to execute behaviour. The sortArray(..) method should not be called from a constructor.